main.py

```diff
@@ -51,6 +53,10 @@ def initialize_session_state():
 #### Update Information ####
 #### ================== ####
 def update():
+    pass
```

Comment:
This function seems unnecessary. I thought maybe it is still here for some weird streamlit functionality, but tested it without and seems to work.

Reply:
alright, will remove then to clean it up